eldar/query.py

```diff
@@ -22,6 +23,23 @@ def preprocess(self, doc):
             doc = doc.lower()
         if self.ignore_accent:
             doc = unidecode(doc)
+
+        # demojise query
+        doc = emoji.demojize(doc, language='en', delimiters=("::__", "__::"))
```

Comment:
We can get rid of the delimiter here as the default delimiter will work for what we are looking to do